src/features/currentTodo.ts

```diff
@@ -1,10 +1,18 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
-const initialState = null as Todo | null;
+type CurrentTodo = Todo | null;
+const initialState = null as CurrentTodo;
 
 export const currentTodoSlice = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: {
+    set(
+      currentTodo: CurrentTodo,
+      { payload }: PayloadAction<CurrentTodo>,
+    ): CurrentTodo {
```

Comment:
The reducer function `set` should have the state as the first parameter and the action as the second parameter. Correct the function signature to: `set(state: CurrentTodo, action: PayloadAction<CurrentTodo>): CurrentTodo { return action.payload; }`.